packages/web/src/scss/tools/_placement.scss

```diff
@@ -102,31 +118,88 @@ $_placement-inverse-map: (
     );
 }
 
-// Function to return an inverse placement name
-// Example: inverse('top') will return 'bottom'
-// Example: inverse('top-start') will return 'bottom'
-// Example: inverse('top-start', true) will return 'bottom-start'
-// Example: inverse('top-start', true, false) will return 'bottom start'
-@function inverse($placement, $main-axis-only: false, $hyphenate: true) {
+// Function to get the cross-axis direction
+// Example: get-cross-axis-direction('left') will return 'vertical'
+// Example: get-cross-axis-direction('bottom') will return 'horizontal'
+@function get-cross-axis-direction($main-axis) {
+    @if $main-axis == 'left' or $main-axis == 'right' {
```

Comment:
Shouldn't be the usage of the ternary operator in this simple function better?

Reply:
can be. Updated.

Reply:
There is only inline `if()` function in Sass, and its legibility is limited… But I have it a few lines below, so why not.